src/lammpsio/dump.py

```diff
@@ -60,14 +119,24 @@ def create(cls, filename, schema, snapshots):
             Path to dump file.
         schema : dict
             Schema for the contents of the file.
-        snapshots : :class:`Snapshot` or list
+        snapshots : `Snapshot` or list
             One or more snapshots to write to the dump file.
 
         Returns
         -------
-        :class:`DumpFile`
+        `DumpFile`
             The object representing the new dump file.
 
+        Example
+        -------
+        A `DumpFile` can be created from a list of snapshots:
+
+        .. skip: next
+
+        .. code-block:: python
+
+            lammpsio.DumpFile.create("atoms.lammpstrj", schema, snapshots)
```

Comment:
This example will not work because there is no schema or snapshots. I don't mind the snapshot part, but there is currently no example showing how to write a schema: this would be the spot to do that.